reward/addressbook_connector_test.go

```diff
@@ -28,11 +28,10 @@ func newTestBlockChain() *blockchain.BlockChain {
 
 func TestAddressBookManager_makeMsgToAddressBook(t *testing.T) {
 	targetAddress := "0x0000000000000000000000000000000000000400" // address of addressBook which the message has to be sent to
-	addressBookManager := newAddressBookManager(newTestBlockChain(), nil)
-	msg, err := addressBookManager.makeMsgToAddressBook()
-	if err != nil {
-		t.Errorf("err has occurred. err : %v", err)
-	} else {
-		assert.Equal(t, targetAddress, msg.To().String())
+	ac := newAddressBookConnector(newTestBlockChain(), nil)
+	msg, err := ac.makeMsgToAddressBook()
+	if !assert.NoError(t, err) {
```

Comment:
(I'm just asking) What is the difference between using `assert.Error` here and using `t.FailNow()`?